src/test/java/com/aws/greengrass/lifecyclemanager/KernelLifecycleTest.java

```diff
@@ -134,6 +178,206 @@ void GIVEN_kernel_WHEN_launch_with_autostart_services_THEN_autostarts_added_as_d
         verify(mockMain, times(2)).addOrUpdateDependency(eq(mockOthers), eq(DependencyType.HARD), eq(true));
     }
 
+    @SuppressWarnings("PMD.CloseResource")
```

Comment:
Which resource? Can you close it instead of suppressing this or use try with resource?

Reply:
It complains about the mocked EZPlugin. Closing the mocked one does not help. Existing test - "GIVEN_kernel_WHEN_launch_with_autostart_services_THEN_autostarts_added_as_dependencies_of_main" has similar issue.